.github/workflows/tests.yml

```diff
@@ -0,0 +1,185 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  linux:
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php: ["5.6", "7.0", "7.1", "7.2", "7.3", "7.4"]
+        stability: ["Stable"]
+        include:
+          - php: "5.6"
+            stability: "Lowest"
+          - php: "8.0"
+            stability: "Ignore"
+
+    name: PHP ${{ matrix.php }} - ${{ matrix.stability }} - Linux
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer:snapshot
+          coverage: none
+
+      - name: Setup problem matchers
```

Comment:
What is this? Can you elaborate or link to some PR when it is run or better - failing a build for good reason?

Reply:
Shows test failures inline on diffs on github's ui.

Reply:
Is it? I don't think I can reproduce: https://github.com/sanmai/phpunit-primer/pull/6/files

Reply:
Yes, that stuff works properly. It's exactly as documented in https://github.com/shivammathur/setup-php#phpunit.

Reply:
Well, the thing is I made [a test to fail intentionally](https://github.com/sanmai/phpunit-primer/pull/6/files#diff-13e3993e50d053aa5752f11a1e9bbf312e598ffa0054b8740260e29b7932d821R15-R18), added [these matchers](https://github.com/sanmai/phpunit-primer/blob/5d8c8bcbc5794b536c2bb9860c08866c21e295dc/.github/workflows/ci.yaml#L68), and there's nothing. How can we verify that this thing actually works?

Reply:
@GrahamCampbell why closing the thread? looks like the question is still open